subprojects/parseq-restli-client/src/main/java/com/linkedin/restli/client/ParSeqRestClient.java

```diff
@@ -119,7 +119,11 @@ public ParSeqRestClient(final RestClient client) {
   @Deprecated
   public <T> Promise<Response<T>> sendRequest(final Request<T> request, final RequestContext requestContext) {
     final SettablePromise<Response<T>> promise = Promises.settable();
-    _client.sendRequest(request, requestContext, new PromiseCallbackAdapter<T>(promise));
+    try {
+      _client.sendRequest(request, requestContext, new PromiseCallbackAdapter<T>(promise));
+    } catch (Exception e) {
```

Comment:
Do you want to catch Exception or throwable?

Reply:
I guess throwable makes more sense. I can update

Reply:
+1 on Throwable

Reply:
+1 on Throwable.